package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
@teclator I think it is generic agreement, as each change is potential risk of failure, so we add to SP2 only what is requested.